rpc/src/main/java/com/strategyobject/substrateclient/rpc/metadata/PalletCollection.java

```diff
@@ -0,0 +1,31 @@
+package com.strategyobject.substrateclient.rpc.metadata;
+
+import lombok.NonNull;
+import lombok.val;
+
+import java.util.Arrays;
+
+public class PalletCollection {
+    private final Pallet[] pallets;
+
+    public PalletCollection(Pallet @NonNull ... pallets) {
+        val maxIndex = Arrays.stream(pallets)
```

Comment:
I have a doubt whether it should support blank spaces.

Reply:
I made a strict contract